cmd/util/feature_flags.go

```diff
@@ -25,10 +25,11 @@ import (
 type FeatureFlag string
 
 const (
-	CDC           FeatureFlag = "CDC"
-	CONFIGURE_URL FeatureFlag = "CONFIGURE_URL"
-	NODE_OP       FeatureFlag = "NODE_OPS"
-	TOOLS         FeatureFlag = "TOOLS"
+	CDC                FeatureFlag = "CDC"
+	CONFIGURE_URL      FeatureFlag = "CONFIGURE_URL"
+	NODE_OP            FeatureFlag = "NODE_OPS"
+	TOOLS              FeatureFlag = "TOOLS"
+	AZURE_CIDR_PROVIDE FeatureFlag = "AZURE_CIDR_PROVIDE"
```

Comment:
nit: CIDR_REQUIRED?

Reply:
If the flag is on, it's not required, but can be provided. The backend then can decide what to do with this value depending on the configuration there.

Reply:
Same as @posriniv I prefer `AZURE_CIDR_ALLOWED`

Reply:
OK, `AZURE_CIDR_ALLOWED` is good.